### PR TITLE
Con 2506 send request new training provider

### PR DESCRIPTION
**Pull request by hkf-tech**

### CODE REVIEW

File: src/SFA.DAS.EmployerCommitmentsV2.Web/RouteValues/RouteNames.cs [hunk 1]
```diff
@@ -11,6 +11,7 @@ public static class RouteNames
         public const string ChangeProviderInform = "changing-training-provider-inform";
         public const string ApprenticeNotStoppedError = "aprentice-not-stopped-error";
         public const string EnterNewTrainingProvider = "enter-new-training-provider-name-or-reference-number";
+        public const string WhoWillEnterNewDetails = "who-eill-enter-new-details";
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/RouteValues/RouteNames.cs:14]: typo

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Validators/SendNewTrainingProviderRequestValidator.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Validators
+{
+    public class SendNewTrainingProviderRequestValidator : AbstractValidator<SendNewTrainingProviderRequest>
+    {
+        public SendNewTrainingProviderRequestValidator()
+        {
+            RuleFor(r => r.AccountHashedId).NotEmpty();
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Validators/SendNewTrainingProviderRequestValidator.cs:14]: do we not need to validate the non-hashed AccountId and ApprenticeshipId? AccountId looks as if it is never used, can it be removed from the request class?

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/SendNewTrainingProviderViewModel.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+﻿using SFA.DAS.Authorization.ModelBinding;
+using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice
+{
+    public class SendNewTrainingProviderViewModel : IAuthorizationContextModel
+    {
+        public string AccountHashedId { get; set; }
+        public long AccountId { get; set; }
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/SendNewTrainingProviderViewModel.cs:9]: AccountId and ApprenticeshipId are never used, can they be removed?

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/SendRequestNewTrainingProvider.cshtml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            @Html.HiddenFor(x => x.NewProviderName)
+            @Html.HiddenFor(x => x.ProviderId)
+            @Html.HiddenFor(x => x.AccountHashedId)
+            @Html.HiddenFor(x => x.ApprenticeshipHashedId)
+            <h1 class="govuk-heading-xl">Are you sure you want to send this request to @Model.NewProviderName?</h1>
+
+            @if (Model.ApprenticeshipStatus != SFA.DAS.CommitmentsV2.Types.ApprenticeshipStatus.Stopped)
+            {
+                <p class="govuk-body das-!-colour-inherit">
+                    Once you send this request, @Model.ApprenticeName's apprenticeship record and payments to @Model.OldProviderName will be stopped
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/SendRequestNewTrainingProvider.cshtml:22]: if the name ends in 's' such as "Jones", should be `'` not `'s`

File: src/SFA.DAS.EmployerCommitmentsV2.Web/RouteValues/RouteNames.cs (hunk 1 above)
Comment (hkf-tech) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/RouteValues/RouteNames.cs]: removed - as it is not used.

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/SendNewTrainingProviderViewModel.cs (hunk 3 above)
Comment (hkf-tech) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/SendNewTrainingProviderViewModel.cs:9]: AccountId & ApprenticeshipId will be required when we create the Cohort

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/SendRequestNewTrainingProvider.cshtml (hunk 4 above)
Comment (hkf-tech) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/SendRequestNewTrainingProvider.cshtml]: Nice catch, never crossed my mind. Made changes for this.

**Review by cofaulco — APPROVED** [review]

**Pull request merged**